app/models/question.rb

```diff
@@ -0,0 +1,35 @@
+class Question < ActiveRecord::Base
+  belongs_to :author, class_name: :User, foreign_key: :author_id
+
+  has_many :comments, as: :commentable
+  has_many :taggings
+  has_many :tags, through: :taggings
+  has_many :votes, as: :voteable
+
+  validates :title, :body, :author_id, presence: true
+
+  def all_tags=(names)
+    self.tags = names.split(",").map do |name|
+      Tag.where(name: name.strip).first_or_create!
+    end
+  end
+
+  def all_tags
+    self.tags.map(&:name).join(", ")
+  end
+
+  def answers
+    self.comments.select{ |comment| comment.is_answer == true }
```

Comment:
Use the database. This is a ruby select so will load all comments and filter the array.
```comments.where(is_answer: true)``` will be more efficient